tests/baselines/reference/libReplacement(libreplacement=false).trace.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Compare to `tests/baselines/reference/libReplacement(libreplacement=true).trace.json` below, which has _1003_ lines of traces for an empty file with `lib=esnext`.

Reply:
Now 1,016 after we added more `lib.d.ts` files 😄